mergeable/src/lib/vendor.d.ts

```diff
@@ -0,0 +1,26 @@
+declare namespace GitHubApi {
```

Comment:
I don't think you need a namespace, just a regular ts file with interfaces exported :)